edge/pkg/eventbus/common/util/common.go

```diff
@@ -78,12 +78,11 @@ func HubClientInit(server, clientID, username, password string) *MQTT.ClientOpti
 		}
 
 		tlsConfig = &tls.Config{
-			RootCAs:            pool,
-			Certificates:       []tls.Certificate{cert},
-			InsecureSkipVerify: false,
+			RootCAs:      pool,
+			Certificates: []tls.Certificate{cert},
 		}
 	} else {
-		tlsConfig = &tls.Config{InsecureSkipVerify: true, ClientAuth: tls.NoClientCert}
```

Comment:
If we disable tls and set `InsecureSkipVerify` to false and didn't set CA cert, it will cause the HTTPS client can't r access to the HTTPS server.